### PR TITLE
Added UI language: Japanese

### PR DESCRIPTION
**Pull request by Cothlory**
Accuracy corrections for English and Chinese version.

### CODE REVIEW

**Review by UlyssesZh — APPROVED** [review]
Good!

File: js/Strings.js [hunk 1]
```diff
@@ -562,8 +563,194 @@ Strings["zh-TW"] = {
...
+	"gameTitle": "ドドド",
+	"ok": "確認",
+	"back": "戻る",
+	"loading": "ロード中...",
+	
+	// title page
+	"browseFiles": "ブラウズファイル",
+	"browseStore": "ストアー",
+	"browseHistory": "閲覧履歴",
+	"preferences": "设置",
```
Comment (Misaka13514) [inline js/Strings.js:581]: consider using `設定` ?

Comment (UlyssesZh) [reply js/Strings.js:581]: Don't know much about Japanese, but you can send a PR. @GloryThunder any ideas?

Comment (Misaka13514) [reply js/Strings.js:581]: and here 3 `browse` are all verbs, but translated in 3 different ways(

Comment (Misaka13514) [reply js/Strings.js:581]: 👀

**Pull request merged**